Makefile

```diff
@@ -147,7 +147,7 @@ olm-push:
 	    QUAY_PASSWORD=${QUAY_PASSWORD} QUAY_REPOSITORY=${QUAY_REPOSITORY} PACKAGE_NAME=${PACKAGE_NAME} ./hack/olm.sh push"
 
 bump-kubevirtci:
-	./hack/bump-kubevirtci.sh
+	hack/dockerized ./hack/bump-kubevirtci.sh
```

Comment:
This is still here? Wasn't this supposed to be removed?

Reply:
We discover that when you don't runt it dockerized the cluster-up digest is different between ci and users, that was the issue with generate job.